Formula/molecule.rb

```diff
@@ -19,7 +20,6 @@ class Molecule < Formula
   depends_on "ansible"
   depends_on "openssl@1.1"
   depends_on "python@3.9"
-  depends_on "six"
```

Comment:
Why change this?

Reply:
I had a question about this one.  I had a problem with Molecule initially where i updated the Molecule package but not the Six package.  This caused an error due to Molecule needing a newer version.  Since Six is a library and not a tool/utility you run, it seemed to fit better as a versioned resource in the formula instead of a brew dependency.  This would also remove any version mismatch issues in the future.  Thoughts on which is best?  Wasn't confident what the preferred method was after looking at the [Python for Formula Authors](https://docs.brew.sh/Python-for-Formula-Authors) docs.

Reply:
Molecule has a dependency on Six, so upgrading Molecule should also upgrade Six.

Reply:
Sorry.... I was remembering incorrectly what the issue I had was.  During the installation of Molecule I got this error:

```
==> Installing molecule dependency: six
==> Pouring six--1.16.0_1.catalina.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink lib/python3.9/site-packages/six.py
Target /usr/local/lib/python3.9/site-packages/six.py
already exists. You may want to remove it:
  rm '/usr/local/lib/python3.9/site-packages/six.py'

To force the link and overwrite all conflicting files:
  brew link --overwrite six

To list all files that would be deleted:
  brew link --overwrite --dry-run six

Possible conflicting files are:
/usr/local/lib/python3.9/site-packages/six.py
```

That's why I moved it from a dependency to resource block.  My question about which approach is better still applies.

Reply:
Yea, that happens when you `pip install` into Homebrew's `site-packages` directory, and not just for `six`. I suggest you do `pip install -U` or install into a virtual env.

Let's keep the `six` dependency -- it avoids duplicating code across different formulae, and duplicating installs of the same Python module on the same machine.